lib/realtime/tenants/authorization.ex

```diff
@@ -57,7 +57,7 @@ defmodule Realtime.Tenants.Authorization do
   def get_authorizations(%Phoenix.Socket{} = socket, db_conn, authorization_context) do
     case get_authorizations(db_conn, authorization_context) do
       %Policies{} = policies -> {:ok, Phoenix.Socket.assign(socket, :policies, policies)}
-      error -> {:error, error}
+      error -> error
```

Comment:
this move may break the return contract: https://github.com/supabase/realtime/pull/1119/files#diff-9542ac1036d63c2de0cf927b9cebf2c4d045ce6388432f56a0ba26ca83920c8bR56